bin/ansible-inventory-grapher

```diff
@@ -105,7 +105,7 @@ def options_parser():
 
 
 def labelescape(name):
-    return name.replace("-", "_").replace(".", "_")
+    return "\"%s\"" % name.replace("-", "_").replace(".", "_")
```

Comment:
Now that you mention it, I'm not really sure about that. Quoting might just be enough. I'll try it soon.